setup/storage/s3.md

```diff
@@ -12,9 +12,14 @@ storage source means that Spinnaker will store all of its persistent data in a
 ## Prerequisites
 
 You need an [AWS Account](https://aws.amazon.com/account/), and an associated
-access key (secret access key & access key id). The service in charge of
-storage does not support STS AssumeRole style auth, so the access key & id are
-required.
+access key (secret access key & access key id).
+
+## Notes
+1. The service in charge of storage does not support STS AssumeRole style auth,
+so the access key & id are required. Ignore `--assume-role` option in the CLI.
```

Comment:
I'd prefer if you made the change in halyard to remove that flag if it's not useful